test/src/unit-conversions.cpp

```diff
@@ -1684,6 +1685,8 @@ TEST_CASE("JSON to enum mapping")
 
         // invalid json -> first enum
         CHECK(cards::kreuz == json("what?").get<cards>());
+
+        CHECK(ordered_json(cards::kreuz) == "kreuz");
```

Comment:
Can you please add a comment mentioning this MR and https://github.com/nlohmann/json/issues/2491 ?

Reply:
Sure

Reply:
Done